.github/workflows/prannouncer.yml

```diff
@@ -0,0 +1,14 @@
+name: "Google Chats PR Announcer"
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 6 * * 1-5"
```

Comment:
I think the following syntax is supported:

```suggestion
    - cron: "0 6 * * MON-FRI"
```

Reply:
Didn't know that, Thanks!